consensus/istanbul/validator/default.go

```diff
@@ -247,6 +247,27 @@ func (valSet *defaultSet) Copy() istanbul.ValidatorSet {
 	return newValSet
 }
 
+// We include the optimization described at https://arxiv.org/pdf/1901.07160.pdf as “PM-6” and
+// discussed in Lemma 22. For values of N=3F for integer F=1,2,3,.. we can tolerate a quorum
+// size one smaller than anticipated by Q = N - F. The intersection of any two sets of Q
+// nodes of N=3F must contain an honest validator.
```

Comment:
Same comment as above
```suggestion
// nodes of N>=3F must contain an honest validator.
```